src/Validators/IntegerValidator.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Portal\Validators;
+
+class IntegerValidator
+{
+    /**
+     * Validates that an integer is within the $min and $max range
+     *
+     * @throws \Exception if number out of range
+     */
+    public static function validateInteger(int $unvalidated, int $min, int $max): int
+    {
+        $options = ["options" => ["min_range" => $min, "max_range" => $max]];
+        if (filter_var($unvalidated, FILTER_VALIDATE_INT, $options) !== false) {
```

Comment:
!== false
is this a double negative?!

Reply:
Thats not equal to false, so its not a double negative.